cmd/dosa/util.go

```diff
@@ -114,16 +117,35 @@ func strToFieldValue(t dosa.Type, s string) (dosa.FieldValue, error) {
 	}
 }
 
-func printResult(res []map[string]dosa.FieldValue) {
-	if len(res) == 0 {
-		fmt.Println("Error: not found")
-		return
+func printResults(results []map[string]dosa.FieldValue) error {
+	const padding = 3
```

Comment:
Should this be a constant? If so maybe it's better to define outside this function (under the imports)?